test/sensitivities/linalg/blas.jl

```diff
@@ -0,0 +1,222 @@
+@testset "sensitivities/blas" begin
+
+let ϵ_abs = 1e-5, ϵ_rel = 1e-4, δ = 1e-6
+
+    # Testing allocating sensitivities for dot.
+    import Base.BLAS.dot
+    let rng = MersenneTwister(123456)
```

Comment:
Perhaps store the seed in some variable somewhere and write `MersenneTwister(tests_seed)` instead.